src/commands/misc_commands/bookmark.ts

```diff
@@ -0,0 +1,354 @@
+import { IPCLogger } from "../../logger";
+import {
+    chunkArray,
+    containsHangul,
+    discordDateFormat,
+    truncatedString,
+} from "../../helpers/utils";
+import {
+    clickableSlashCommand,
+    getInteractionValue,
+    localizedAutocompleteFormat,
+    searchArtists,
+    sendDeprecatedTextCommandMessage,
+    sendPaginationedEmbed,
+    tryAutocompleteInteractionAcknowledge,
+    tryCreateInteractionErrorAcknowledgement,
+} from "../../helpers/discord_utils";
+import { getEmojisFromSongTags } from "../../helpers/game_utils";
+import Eris from "eris";
+import GameRound from "../../structures/game_round";
+import LocaleType from "../../enums/locale_type";
+import MessageContext from "../../structures/message_context";
+import QueriedSongWithBookmarkDate from "../../structures/queried_song_with_bookmark_date";
+import SongSelector from "../../structures/song_selector";
+import State from "../../state";
+import _ from "lodash";
+import dbContext from "../../database_context";
+import i18n from "../../helpers/localization_manager";
+import type { DefaultSlashCommand } from "../interfaces/base_command";
+import type { EmbedOptions } from "eris";
+import type BaseCommand from "../interfaces/base_command";
+import type CommandArgs from "../../interfaces/command_args";
+import type HelpDocumentation from "../../interfaces/help";
+import type MatchedArtist from "src/interfaces/matched_artist";
+
+const COMMAND_NAME = "bookmark";
+const logger = new IPCLogger(COMMAND_NAME);
+
+// eslint-disable-next-line import/no-unused-modules
+export default class BookmarkCommand implements BaseCommand {
```

Comment:
'bookmark' sounds like an action to create a bookmark, not an action to list bookmarks. Should change every reference of 'bookmark' to 'bookmarks'